package.json

```diff
@@ -37,6 +37,7 @@
     "feed-read": "0.0.1",
     "fs-nextra": "^0.3.4",
     "gifwrap": "^0.7.5",
+    "imgur": "^0.2.1",
```

Comment:
No. I specifically removed this dependency to implement a custom one in GAB that was
A. class based
B. Not using request.

Reply:
:thonk:

Should've told me earlier.